pnpm-lock.yaml

```diff
@@ -2249,7 +2249,7 @@ importers:
         version: 5.9.0
       tsup:
         specifier: ^8.4.0
-        version: 8.4.0(postcss@8.5.3)(tsx@4.19.4)(typescript@5.8.3)
+        version: 8.4.0(postcss@8.5.3)(tsx@4.19.4)(typescript@5.8.3)(yaml@2.7.1)
```

Comment:
Whatever caused this resolution to change added a second copy of tsup even though it's the same version. We should probably run `pnpm dedupe` again. Probably easiest in a follow up PR unless the diff is very small in this same PR.